rdkit/Chem/SaltRemover.py

```diff
@@ -171,16 +228,18 @@ def _applyPattern(m, salt, notEverything):
       return mol
```

Comment:
OK. I will implement the changes beginning of next week. Many thanks for your comments. I will write this test after having understood the test infrastructure of this project.
By the way, **SaltRemover** does not fully reflect the capabilities of this class, right? Actually, this class is able to remove solvents as well, is not it?